x/bank/simulation/operations.go

```diff
@@ -93,6 +93,155 @@ func SimulateMsgSend(
 	}
 }
 
+// GenerateMsgSend generates a randomized single send transaction.
+func GenerateMsgSend(r *rand.Rand, ctx sdk.Context, clientCtx client.Context, chainID string, txGen client.TxConfig, moniker string, to, from simtypes.Account, bk keeper.Keeper, ak types.AccountKeeper) sdk.Tx {
```

Comment:
Minor nit: please the name "rng" for Random Number Generator instead of "r" which is idiomatically for io.Reader

Reply:
Same as below. The old simulator unfortunately uses `r` everywhere.